.editorconfig

```diff
@@ -0,0 +1,10 @@
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+indent_size = 4
+indent_style = space

```

Comment:
The rest of the project uses tab-based indentation, as Go is officially supposed to: https://golang.org/doc/effective_go.html#formatting

Reply:
Good catch, I applied the change.